{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"level":"error","message":"‚ùå Database connection failed: Access denied for user 'sql8768840'@'104.28.251.98' (using password: YES)","sqlMessage":"Access denied for user 'sql8768840'@'104.28.251.98' (using password: YES)","sqlState":"28000","stack":"Error: Access denied for user 'sql8768840'@'104.28.251.98' (using password: YES)\n    at Packet.asError (/Users/terabyte/Desktop/Assessment/lendsqr-wallet/node_modules/mysql2/lib/packets/packet.js:740:17)\n    at ClientHandshake.execute (/Users/terabyte/Desktop/Assessment/lendsqr-wallet/node_modules/mysql2/lib/commands/command.js:29:26)\n    at Connection.handlePacket (/Users/terabyte/Desktop/Assessment/lendsqr-wallet/node_modules/mysql2/lib/base/connection.js:475:34)\n    at PacketParser.onPacket (/Users/terabyte/Desktop/Assessment/lendsqr-wallet/node_modules/mysql2/lib/base/connection.js:93:12)\n    at PacketParser.executeStart (/Users/terabyte/Desktop/Assessment/lendsqr-wallet/node_modules/mysql2/lib/packet_parser.js:75:16)\n    at Socket.<anonymous> (/Users/terabyte/Desktop/Assessment/lendsqr-wallet/node_modules/mysql2/lib/base/connection.js:100:25)\n    at Socket.emit (node:events:519:28)\n    at Socket.emit (node:domain:488:12)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)","timestamp":"2025-03-29T12:01:07.399Z"}
